elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -11,9 +11,10 @@
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     required_attribs = ['answers-name']
-    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline']
+    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline', 'enable-nota']
```

Comment:
Can we name the attribute `none-of-the-above` to be more consistent with the others?

Reply:
updated 👍